### PR TITLE
VariableSettings: fix tool-tip hyperlink color

### LINKED ISSUE #36122: Make hyperlinks blue in info tooltips
When I hover over an (i) info tooltip containing a hyperlink, I want the hyperlink to render and blue and underlined, so that I know I can click on it. In light mode, I'd like to be able to see it.

An example of a tooltip with a hyperlink is the Grafana 7.4 Variables config screen, on the Regex field.

Current state:
Light mode:

Dark mode:

### PR DESCRIPTION
**Pull request by tharun208**
**What this PR does / why we need it**: Right now, it is difficult to recognize the tool-tip in the variable's setting `regex` field due to the missing color or indication of the hyperlink.

**Which issue(s) this PR fixes**: This PR fixed this by adding `color` to the hyperlink.

Fixes #36122

**Special notes for your reviewer**:

### CODE REVIEW

**Review by tskarhed — CHANGES_REQUESTED** [review]
I still don't think this offers enough contrast. 

Screenshot from 2021-06-29 10-09-17

**Comment by tharun208** [comment]
@tskarhed, can I use a different class ?. the class I referred to is used by other hyperlink

**Comment by tskarhed** [comment]
> @tskarhed, can I use a different class ?. the class I referred to is used by other hyperlink

I'm not sure. Hyperlinks aren't typically used in tooltips, that's why it looks off. Maybe moving the link outside of the tooltip could be an option.

**Comment by tharun208** [comment]
> > @tskarhed, can I use a different class ?. the class I referred to is used by other hyperlink
> 
> I'm not sure. Hyperlinks aren't typically used in tooltips, that's why it looks off. Maybe moving the link outside of the tooltip could be an option.

@tskarhed, instead of the tooltip, can use `Modal`?

**Comment by tharun208** [comment]
@tskarhed found that we are using 2 different versions of `external-link-color` for the dark and white themes ( https://github.com/grafana/grafana/blob/main/public/sass/_variables.dark.generated.scss#L120). can I update it to match with the white theme external link color?

**Comment by Ijin08** [comment]
The problem with this solution is that the link color in the light theme is suppose to be accessible on light backgrounds. The tooltip background however is dark. 
I think the pr solves the wrong problem. The issue is that the tooltip is not designed with hyper links in mind. The example showed in the issue uses the inline form styles used in query editor when it should be the vertical form styles. Then the tooltip would be replaced with a description that then could use the blue link color. 
The issue would still be there if we use links in tooltips in the current query editors. But then we would have to change background color to tooltip.

**Comment by tharun208** [comment]
@Ijin08 any possible ux workaround for this ?

**Comment by Ijin08** [comment]
@tharun208 
For now this would be the best solution:
Screenshot 2021-06-30 at 12 22 59
Adding a underline to the link and using the maxContrast color for the text color.

**Comment by tharun208** [comment]
sorry for the force push 🙏

File: public/app/features/variables/query/QueryVariableEditor.tsx [hunk 1]
```diff
@@ -190,6 +190,10 @@ export class QueryVariableEditorUnConnected extends PureComponent<Props, State>
                   Optional, if you want to extract part of a series name or metric node segment. Named capture groups
                   can be used to separate the display text and value (
                   <a
+                    className={css`
+                      color: #fff;
+                      text-decoration: underline;
+                    `}
```
Comment (torkelo) [inline public/app/features/variables/query/QueryVariableEditor.tsx:196]: This needs to be fixed in https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Tooltip/_Tooltip.scss  not inline for this specific link

File: packages/grafana-ui/src/components/Tooltip/_Tooltip.scss [hunk 2]
```diff
@@ -18,6 +18,10 @@ $popper-margin-from-ref: 5px;
   em {
     color: lighten($textColor, 20%);
   }
+  a {
+    color: #fff;
```
Comment (torkelo) [inline packages/grafana-ui/src/components/Tooltip/_Tooltip.scss:22]:
```suggestion
    color: $white;
```

Comment (tharun208) [reply packages/grafana-ui/src/components/Tooltip/_Tooltip.scss]: addressed

Comment (tharun208) [reply packages/grafana-ui/src/components/Tooltip/_Tooltip.scss]: addressed

**Pull request merged**